src-java/northbound-service/northbound/src/main/java/org/openkilda/northbound/service/LinkService.java

```diff
@@ -106,13 +108,15 @@
      */
     CompletableFuture<List<LinkDto>> deleteLink(LinkParametersDto linkParameters, boolean force);
 
-    /**
-     * Update "Enable BFD" flag.
-     *
-     * @param link link parameters.
-     * @return updated link.
-     */
-    CompletableFuture<List<LinkDto>> updateLinkEnableBfd(LinkEnableBfdDto link);
+    CompletableFuture<List<LinkDto>> bfdPropertiesWrite(
```

Comment:
Rename methods to `write/read/deleteBfdProperties`.

Reply:
done